docs/Project.toml

```diff
@@ -1,7 +1,5 @@
 [deps]
-AbstractAlgebra = "c3fe647b-3220-5bb0-a1ea-a7954cac585d"
```

Comment:
I think this is needed for the doctests from AbstractAlgebra (and hence the docs CI job is failing). To wit, we have this kind of thin in AA a lot:
````
```jldoctest; setup = :(using AbstractAlgebra)
...
````
and that fails now.